readme.md

```diff
@@ -134,16 +134,6 @@ $app->routeMiddleware([
     'cors' => \Barryvdh\Cors\HandleCors::class,
 ]);
 ```
-
-## Common problems and errors (Pre Laravel 5.3)
```

Comment:
This actually still can be a problem, but only when using route middleware instead of global middleware.